src/middlewares/mail.js

```diff
@@ -48,7 +50,10 @@ export const SendVerificationEmail = async (req, res, next) => {
       return res.status(403).json({ status: 403, error: response.message });
     }
     if (response.message === 'queryA ECONNREFUSED smtp.gmail.com') {
-      return res.status(409).json({ status: 409, error: response.message });
+      return res.status(409).json({ status: 409, error: 'Network error occured, please check your network' });
```

Comment:
Kindly use a different status code. 409 status code is for conflict not network related issues

Reply:
done